dashboard/modules/job/job_manager.py

```diff
@@ -463,24 +477,6 @@ def _get_actor_for_job(self, job_id: str) -> Optional[ActorHandle]:
 
     async def _monitor_job(
         self, job_id: str, job_supervisor: Optional[ActorHandle] = None
-    ):
-        """Monitors the specified job until it enters a terminal state.
```

Comment:
why is it deleted?

Reply:
Ah good catch, I accidentally deleted the docstring.